contents/blog/best-fathom-alternatives.mdx

```diff
@@ -0,0 +1,468 @@
+- **Typical users:** Engineers and product teams
+- **Typical customers:** Mid-size B2Bs and startups
+
+<ProductScreenshot
+  imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_01_at_11_39_58_2x_5dff6baf53.png"
+  imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_01_at_11_40_26_2x_76efad2201.png"
+  classes="rounded"
+  alt="PostHog web analytics dashboard"
+/>
+
+### What is PostHog?
+
+[PostHog](/) (that's us 👋) is an open source platform combining web analytics, product analytics, session replay, A/B testing, feature flags, and user surveys into one product. This means it's not only an alternative to Fathom but also tools like [Mixpanel](/blog/posthog-vs-mixpanel), [LaunchDarkly](/blog/posthog-vs-launchdarkly), and [Hotjar](/blog/posthog-vs-hotjar).
+
+According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of October 2024, 5,330 of the top 1 million websites deploy PostHog. Significantly more than the 1,832 using Fathom.
+
+#### Key features
+
+- 📊 **Web analytics:** Monitor your web traffic by automatically capturing and calculating metrics like visitors, pageviews, session duration, and bounce rate. See the sources, entry and exit paths, channels, and more.
+
+- 📈 **Product analytics:** Custom trends, funnels, user paths, retention analysis, and segment user cohorts. Also, direct [SQL querying](/docs/product-analytics/sql) for full customization.
+
+- 📺 **Session replays:** View exactly how users are using your site. Includes event timelines, console logs, network activity, and 90-day data retention.
+
+- 💬 **Surveys:** Target surveys by event or user properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.
+
+- 🧪 **A/B tests:** Optimize your app and website with up to nine test variations and track impact on primary and secondary metrics. Automatically calculate test duration, sample size, and statistical significance.
+
+### How does PostHog compare to Fathom?
+
+Fathom focuses entirely on web analytics while PostHog includes web analytics with a whole suite of tools for analyzing and improving your site and app. This means PostHog has a wider feature set while still offering the privacy-prerserving options Fathom prides itself on. PostHog is also open source and provides a generous free tier, neither of which is true of Fathom. 
+
+<ComparisonTable column1="PostHog" column2="Fathom">
+  <ComparisonRow column1={true} column2={false} feature="Free tier" description="Free forever option, no credit card required" />
+  <ComparisonRow column1={true} column2={true} feature="Web analytics" description="Pre-built dashboard of aggregate traffic, sources, campaigns, and session metrics" />
+  <ComparisonRow column1={true} column2={"Events only"} feature="Custom events" description="Manually capture custom events and properties wherever they happen" />
```

Comment:
Would it be better / clearer to have separate fields for custom events and properties?